Formula/mypy.rb

```diff
@@ -2,8 +2,8 @@ class Mypy < Formula
   desc "Experimental optional static type checker for Python"
   homepage "http://www.mypy-lang.org/"
   url "https://github.com/JukkaL/mypy.git",
-    :tag => "v0.4.2",
-    :revision => "b8a2df91707bc3d5c9c3677c8017b614e41d017a"
+      :tag => "v0.4.3",
```

Comment:
>  (unless someone is willing to do the work to customize it) 

Honestly, I think this is where we need to end up, similar to [cask](https://github.com/caskroom/rubocop-cask). A custom Rubocop that works for Homebrew, that we can write our own desires into. It's just finding the time for any of us to take the steps towards getting that done that's a pain.